metadata/window-rules.xml

```diff
@@ -4,10 +4,10 @@
 		<_short>Window Rules</_short>
 		<_long>A plugin to apply specific commands to windows by using various criteria.</_long>
 		<category>Window Management</category>
-		<option name="rules" type="dynamic-list">
+		<option name="rules" type="dynamic-list" type-hint="plain">
 			<_short>Window rules</_short>
 			<_long>Apply specific commands to windows by using various criteria</_long>
-			<entry prefix="rule_" type="string">
```

Comment:
Reason for this change?

Reply:
Two reasons:
1. It does not make sense to have the prefix: single entry.
2. The current code does not care for the prefix.

PS: The current code does not treat `rules` as a compound option at all - I just looked at the [code](https://github.com/WayfireWM/wayfire/blob/master/plugins/window-rules/window-rules.cpp#L199), but as a bunch of normal options.

Reply:
Makes sense.